bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ThingManager.java

```diff
@@ -512,7 +525,7 @@ public Void call() throws Exception {
                 }
             } else {
                 logger.debug("Cannot notify handler about updated thing {}, because thing is not initialized "
```

Comment:
This should probably be changed to "handler is not initialized", see https://github.com/eclipse/smarthome/pull/2599#discussion_r91108335.